pacehrh/R/pace_task_manage.R

```diff
@@ -1,21 +1,3 @@
-.taskColumnNames <- c(
-  "Indicator",
-  "CommonName",
-  "ClinicalOrNon",
-  "ClinicalCat",
-  "ServiceCat",
-  "RelevantPop",
-  "Geography",
```

Comment:
this "Geography" column is not in either optional or required but this column is still being checked by lint tool, we should remove that reference if it is not needed
https://github.com/InstituteforDiseaseModeling/PACE-HRH/blob/10ba0ddbf78d4495fd4302c0b9ccfa5d6d4d2b36/pacehrh/R/pace_lint.R#L298

Reply:
Sorted!